onnxmltools/convert/sklearn/operator_converters/FeatureUnion.py

```diff
@@ -0,0 +1,44 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+from ....proto import onnx_proto
+from ...common._apply_operation import apply_abs, apply_cast, apply_mul, apply_reshape, apply_sub
+from ...common._registration import get_converter, register_converter
+from .._parse import sklearn_operator_name_map
+import numpy as np
+from sklearn.decomposition import PCA
+from sklearn.decomposition import TruncatedSVD
+
+class Output:
+    def __init__(self, full_name):
+        self.full_name = full_name
+
+class Oper:
+    def __init__(self, model, inputs, op_type, input_full_names, output_full_name):
+        self.raw_operator = model
+        self.inputs = inputs
+        self.input_full_names = input_full_names
+        self.type = op_type
+        self.outputs = [Output(output_full_name)]
+
+def convert_sklearn_feature_union(scope, operator, container):
```

Comment:
This converter needs document to explain the overall idea. Please also add inline comments for non-trivial details.